QMChatService/QMChatService/QMChatService.m

```diff
@@ -1160,6 +1160,15 @@ - (void)sendAttachmentMessage:(QBChatMessage *)attachmentMessage
     }
 }
 
+- (void)sendStickerMessage:(QBChatMessage *)message
+                  toDialog:(QBChatDialog *)dialog
+             saveToHistory:(BOOL)saveToHistory
+             saveToStorage:(BOOL)saveToStorage
+                completion:(QBChatCompletionBlock)completion
+{
+    [self sendMessage:message type:QMMessageTypeSticker toDialog:dialog saveToHistory:saveToHistory saveToStorage:saveToStorage completion:completion];
```

Comment:
AFAIK message, that was received by this method), should contain stickerID in it. I suggest you to add assert for it to be nonnull, as some of developers may experiencing unexpected problems if they will try to send sticker message with nil sticker ID.
Example: NSAssert(message.stickerID != nil, @"Sticker ID required!");